src/ape/utils/basemodel.py

```diff
@@ -201,7 +201,7 @@ def __getattr__(self, item):
     def __getitem__(self, item):
         return self._callback(item)
 
-    def get(self, item):
+    def get(self, item):  # type: ignore[override]
```

Comment:
basically consider this an interface and we are never using any other kwargs in the context it gets called